IRremoteESP8266.cpp

```diff
@@ -108,10 +108,9 @@ void IRsend::begin() {
 }
 
 // Generic method for sending data that is common to most protocols.
-// Default to transmitting the Most Significant Bit (MSB) first.
 void IRsend::sendData(unsigned int onemark, unsigned long onespace,
                       unsigned int zeromark, unsigned long zerospace,
-                      uint32_t data, uint8_t nbits, bool MSBfirst = true) {
```

Comment:
checkout "git rebase -i". after fixing commits locally you push again but with -f (force) to the same branch and github will update the pull request.